src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an
+ * example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file]
+ * (./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+  const makeArray = sourceString.replace(/(\n|\r)/gm, '')
+    .split(':').map(item => item.trim()).join(':').split(';')
+    .map(item => item.trim()).filter(item => item !== '');
```

Comment:
Instead of using multiple .map() and .filter() calls, you can use a single .reduce() function to create the cssObject.